<<<<<<< HEAD
ERROR - 2023-09-07 03:06:23 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'casas' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'casas', 3306, '', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(48): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'casas' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\casas\app\Controllers\Home.php(48): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-09-07 03:06:23 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'casas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casas`
JOIN `casausuario` ON `casas`.`id` = `casausuario`.`id`', [], false)
 3 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(48): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 6 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 03:08:02 --> mysqli_sql_exception: Unknown column 'casausuario.id' in 'on clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
=======
ERROR - 2023-09-07 18:37:16 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
<<<<<<< HEAD
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(48): CodeIgniter\Model->__call('get', Array)
=======
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:08:02 --> Unknown column 'casausuario.id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casas`
JOIN `casausuario` ON `casas`.`id` = `casausuario`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casas`
JOIN `casausuario` ON `casas`.`id` = `casausuario`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casas`
JOIN `casausuario` ON `casas`.`id` = `casausuario`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casas`
JOIN `casausuario` ON `casas`.`id` = `casausuario`.`id`', [], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(48): CodeIgniter\Model->__call('get', [])
=======
ERROR - 2023-09-07 18:37:16 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:37:16 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:08:59 --> Undefined variable $query
=======
CRITICAL - 2023-09-07 18:37:16 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:40:16 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:40:16 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:40:17 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:40:17 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:40:41 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:40:41 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t2`.`id` = `t1`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:41:47 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:41:47 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:41:57 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:41:57 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t1` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t1` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t1` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t1` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 18:42:01 --> mysqli_sql_exception: Unknown column 't1.idCliente' in 'where clause' in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\casas\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\casas\system\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\casas\app\Controllers\Home.php(51): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\casas\system\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
#7 C:\xampp\htdocs\casas\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\casas\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\casas\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-07 18:42:01 --> Unknown column 't1.idCliente' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `casausuario`
JOIN `casas` `t2` ON `t1`.`id` = `t2`.`id`
WHERE `t1`.`idCliente` = :t1.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(836): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Home.php(51): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 18:43:24 --> Undefined variable $query
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
in APPPATH\Views\welcome_message.php on line 4.
 1 APPPATH\Views\welcome_message.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $query', 'C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php', 4)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(66): view('welcome_message', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:09:30 --> Undefined variable $data
in APPPATH\Views\welcome_message.php on line 26.
 1 APPPATH\Views\welcome_message.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php', 26)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(66): view('welcome_message', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:09:42 --> Cannot use object of type mysqli as array
in APPPATH\Views\welcome_message.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 4 APPPATH\Controllers\Home.php(66): view('welcome_message', [...])
 5 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 6 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:10:08 --> Cannot use object of type mysqli as array
in APPPATH\Views\welcome_message.php on line 29.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 4 APPPATH\Controllers\Home.php(66): view('welcome_message', [...])
 5 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 6 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:10:54 --> Cannot use object of type mysqli as array
in APPPATH\Views\welcome_message.php on line 31.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 4 APPPATH\Controllers\Home.php(66): view('welcome_message', [...])
 5 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 6 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:11:21 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::array()
in APPPATH\Controllers\Home.php on line 48.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:12:03 --> Undefined variable $query
in APPPATH\Controllers\Home.php on line 48.
 1 APPPATH\Controllers\Home.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $query', 'C:\\xampp\\htdocs\\casas\\app\\Controllers\\Home.php', 48)
=======
CRITICAL - 2023-09-07 18:43:37 --> Call to undefined method App\Models\casasModel::getResultArray
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Home.php(46): CodeIgniter\Model->__call('getResultArray', [])
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:12:29 --> view(): Argument #2 ($data) must be of type array, string given, called in C:\xampp\htdocs\casas\app\Controllers\Home.php on line 66
in SYSTEMPATH\Common.php on line 1164.
 1 APPPATH\Controllers\Home.php(66): view('welcome_message', '[]')
=======
CRITICAL - 2023-09-07 18:43:37 --> Call to undefined method App\Models\casasModel::getResultArray
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Home.php(46): CodeIgniter\Model->__call('getResultArray', [])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 18:44:00 --> view(): Argument #2 ($data) must be of type array, CodeIgniter\Database\MySQLi\Result given, called in C:\xampp\htdocs\casas\app\Controllers\Home.php on line 66
in SYSTEMPATH\Common.php on line 1164.
 1 APPPATH\Controllers\Home.php(66): view('welcome_message', Object(CodeIgniter\Database\MySQLi\Result))
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:13:26 --> Undefined variable $data
in APPPATH\Controllers\Home.php on line 66.
 1 APPPATH\Controllers\Home.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\casas\\app\\Controllers\\Home.php', 66)
=======
CRITICAL - 2023-09-07 18:44:01 --> view(): Argument #2 ($data) must be of type array, CodeIgniter\Database\MySQLi\Result given, called in C:\xampp\htdocs\casas\app\Controllers\Home.php on line 66
in SYSTEMPATH\Common.php on line 1164.
 1 APPPATH\Controllers\Home.php(66): view('welcome_message', Object(CodeIgniter\Database\MySQLi\Result))
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
CRITICAL - 2023-09-07 03:30:34 --> Undefined variable $data
in APPPATH\Views\welcome_message.php on line 25.
 1 APPPATH\Views\welcome_message.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php', 25)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\casas\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(50): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 03:31:11 --> Undefined variable $data
in APPPATH\Controllers\Home.php on line 50.
 1 APPPATH\Controllers\Home.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\casas\\app\\Controllers\\Home.php', 50)
=======
CRITICAL - 2023-09-07 18:44:02 --> view(): Argument #2 ($data) must be of type array, CodeIgniter\Database\MySQLi\Result given, called in C:\xampp\htdocs\casas\app\Controllers\Home.php on line 66
in SYSTEMPATH\Common.php on line 1164.
 1 APPPATH\Controllers\Home.php(66): view('welcome_message', Object(CodeIgniter\Database\MySQLi\Result))
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
<<<<<<< HEAD
=======
CRITICAL - 2023-09-07 18:44:15 --> view(): Argument #2 ($data) must be of type array, CodeIgniter\Database\MySQLi\Result given, called in C:\xampp\htdocs\casas\app\Controllers\Home.php on line 67
in SYSTEMPATH\Common.php on line 1164.
 1 APPPATH\Controllers\Home.php(67): view('welcome_message', Object(CodeIgniter\Database\MySQLi\Result))
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 18:44:34 --> Call to undefined method App\Models\casasModel::getArray
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Home.php(46): CodeIgniter\Model->__call('getArray', [])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 18:45:05 --> Call to undefined method App\Models\casasModel::getResultArray
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Home.php(46): CodeIgniter\Model->__call('getResultArray', [])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-07 20:05:13 --> Call to undefined method App\Models\casasModel::getResultArray
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Home.php(46): CodeIgniter\Model->__call('getResultArray', [])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->usuario('1')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-07 20:42:40 --> mysqli_sql_exception: Table 'cliente' already exists in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\casas\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\casas\app\Database\Migrations\2023-08-23-174320_CreateClienteTable.php(22): CodeIgniter\Database\Forge->createTable('cliente')
#5 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateClienteTable->up()
#6 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\casas\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\casas\system\CLI\BaseCommand.php(115): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\casas\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\casas\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\casas\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-09-07 20:43:38 --> mysqli_sql_exception: Table 'cliente' already exists in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\casas\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\casas\app\Database\Migrations\2023-08-23-174320_CreateClienteTable.php(22): CodeIgniter\Database\Forge->createTable('cliente')
#5 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateClienteTable->up()
#6 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\casas\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\casas\system\CLI\BaseCommand.php(115): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\casas\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\casas\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\casas\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-09-07 20:56:23 --> mysqli_sql_exception: Table 'cliente' already exists in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\casas\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\casas\app\Database\Migrations\2023-08-23-174320_CreateClienteTable.php(22): CodeIgniter\Database\Forge->createTable('cliente')
#5 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateClienteTable->up()
#6 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\casas\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\casas\system\CLI\BaseCommand.php(115): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\casas\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\casas\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\casas\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-09-07 20:57:13 --> mysqli_sql_exception: Table 'cliente' already exists in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\casas\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\casas\app\Database\Migrations\2023-08-23-174320_CreateClienteTable.php(22): CodeIgniter\Database\Forge->createTable('cliente')
#5 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateClienteTable->up()
#6 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\casas\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\casas\system\CLI\BaseCommand.php(115): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\casas\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\casas\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\casas\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-09-07 21:00:02 --> mysqli_sql_exception: Table 'cliente' already exists in C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\casas\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\casas\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\casas\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\casas\app\Database\Migrations\2023-08-23-174320_CreateClienteTable.php(22): CodeIgniter\Database\Forge->createTable('cliente')
#5 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateClienteTable->up()
#6 C:\xampp\htdocs\casas\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\casas\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\casas\system\CLI\BaseCommand.php(115): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\casas\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\casas\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\casas\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\casas\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
>>>>>>> 57f317e34b336be5aa01d3ed5a94966171ecd151
